frontend/src/components/StatsPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Container, Button, Table } from "reactstrap";
+import { PlayerStats } from "../models/stats.model";
+import { RootSchema } from "../store/root-schema";
+import { Dispatch } from "../store";
+import { changeView } from "../store/actions";
+import { connect } from "react-redux";
+
+interface Props {
+  username: string;
+  stats: PlayerStats;
+  dispatch: Dispatch;
+}
+
+const StatsPage: React.FunctionComponent<Props> = ({
+  username,
+  stats,
+  dispatch,
+}: Props) => {
+  const back = () => {
+    dispatch(changeView("login"));
+  };
+  const gameSuccessRatio =
```

Comment:
I'd display it as 20/68, and maybe next to it percentage, as well.
0.2678912378 looks kinda ugly.

Reply:
Also... Are we 120% sure, that `stats.playedGames`, and `stats.playedRounds` are never 0?